app/Kconfig

```diff
@@ -248,6 +248,39 @@ config ZMK_RGB_UNDERGLOW_ON_START
 #ZMK_RGB_UNDERGLOW
 endif
 
+menuconfig ZMK_BACKLIGHT
+	bool "LED backlight"
+	select PWM
+	select LED
+	select LED_PWM
+
+if ZMK_BACKLIGHT
+
+config ZMK_BACKLIGHT_LEVELS
+	int "Number of brightness levels"
+	range 2 100
+	default 6
+
+config ZMK_BACKLIGHT_LEVEL_START
+	int "Default brightness level"
+	range 1 ZMK_BACKLIGHT_LEVELS
+	default ZMK_BACKLIGHT_LEVELS
+
+config ZMK_BACKLIGHT_ON_START
```

Comment:
Ignore this. I forgot to update it after seeing parts of the code below. Doing this would make it so if you started with backlighting off and toggled it on, it would go to the initial brightness level instead of a probably more reasonable brightness.